css/header-footer.css

```
@@ -231,24 +212,26 @@ footer {
     text-align: center;
 
     font-family: 'Ubuntu', sans-serif;
-    letter-spacing: 1px;
     color: rgb(241, 246, 249);
 
-    background-color: rgb(243, 74, 82);
+    background-color: #7a82bf;
 }
 
 #footer-text {
     display: inline;
 }
 
+#email {
+    color: #f11d28;
```

```suggestion
    color: rgba(241, 29, 40, 0.8);
```
I would do it a little less bright, so I added the higher opacity, let me know what you think